packages/reporters/dev-server/src/Server.js

```diff
@@ -196,6 +196,12 @@ export default class Server {
         });
 
       let indexFilePath = null;
+      let {pathname: reqURL} = url.parse(req.originalUrl || req.url);
```

Comment:
Looks like previously it only looked at `req.url`, is there a reason why you added `originalUrl`?

I remember that we had problems with `req.originalUrl` in the past if you use [the proxy in front of the dev server](https://parceljs.org/features/development/#api-proxy).

Reply:
Just wanted to make sure we are using the same pathname everywhere like [here](https://github.com/parcel-bundler/parcel/pull/9347/files#diff-d23c10b6a0a9c2979d27bca0bba2e7c0eacc9f4fdc602d2c1d4681df75a95776L140).